medical_lab/models/medical_lab_test_type.py

```diff
@@ -0,0 +1,36 @@
+# coding=utf-8
+# Copyright 2012-2013 Federico Manuel Echeverri Choux
+# Copyright 2016 LasLabs Inc.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
+
+
+class MedicalLabTestType(models.Model):
+    _name = "medical.lab.test.type"
+    _description = "Medical Lab Test Types"
+
+    name = fields.Char(
+        help='Name of test type, such as X-Ray, Hemogram, Biopsy, etc.',
+        required=True,
+    )
+    code = fields.Char(
+        size=128,
+        help='Short name or code for test type.',
+        required=True,
+    )
+    description = fields.Text()
+    product_id = fields.Many2one(
```

Comment:
Here just list all the products, for this case can you filter only the types of products that are services or another that is defined?